checkov/cloudformation/checks/resource/aws/SagemakerDataQualityJobDefinitionEncryption.py

```diff
@@ -0,0 +1,26 @@
+from typing import List
+
+from checkov.cloudformation.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.consts import ANY_VALUE
+from checkov.common.models.enums import CheckCategories
+
+
+class SagemakerDataQualityJobDefinitionEncryption(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure Amazon Sagemaker Data Quality Job uses KMS to encrypt model artifacts"
+        id = "CKV_AWS_367"
+        supported_resources = ["AWS::SageMaker::DataQualityJobDefinition"]
+        categories = [CheckCategories.ENCRYPTION]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def get_inspected_key(self):
+        return 'Properties/DataQualityJobOutputConfig/KmsKeyId'
+
+    def get_expected_value(self):
+        return ANY_VALUE
+
+    def get_evaluated_keys(self) -> List[str]:
```

Comment:
I think we don't need to implement this function if we already have `get_inspected_key`.
I'll appreciate if you remove it and check if the tests still pass (also if this is the case do it for the other policies as well)

Reply:
removed it